<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn</class>
 <widget class="QDialog" name="SignIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="setBtn">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>52</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signUpBtn">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>85</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="forgetPasswordBtn">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>118</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>忘记密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>85</y>
     <width>43</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>86</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="signinBtn">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>151</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwdLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>118</y>
     <width>30</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>密码:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLineEdit">
   <property name="geometry">
    <rect>
     <x>108</x>
     <y>119</y>
     <width>125</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitBtn">
   <property name="geometry">
    <rect>
     <x>239</x>
     <y>151</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="showLabel">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>52</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>gridLayoutWidget</zorder>
  <zorder>passwdLabel</zorder>
  <zorder>nameLabel</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>setBtn</zorder>
  <zorder>signUpBtn</zorder>
  <zorder>forgetPasswordBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
